coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java

```diff
@@ -59,9 +60,8 @@ public void setUp()
     @Test
     public void getFoodItemsForOrderTest()
     {
-        List<String> expected = List.of("EVERYTHING bagel with HERB_GARLIC_CREAM_CHEESE",
-                "CHOCOLATE_CHIP cookie", "GLAZED donut");
-        assertEquals(expected, coffeeShopOrder.getFoodItemsForOrder());
+        List<String> expected = List.of("CHOCOLATE_CHIP cookie", "EVERYTHING bagel with HERB_GARLIC_CREAM_CHEESE", "GLAZED donut");
+        assertThat(coffeeShopOrder.getFoodItemsForOrder()).hasSameElementsAs(expected);
```

Comment:
It is possible to use a `Bag` from Eclipse Collections to test that two Lists contain the same elements without requiring the same order. Since Eclipse Collections is already a dependency in other modules in this repo, I would prefer `Bag` be used here to test element only equality instead of adding a new library dependency.

The code would look as follows:

`assertEquals(Bags.mutable.of(expected), Bags.mutable.of(coffeeShopOrder.getFoodItemsForOrder()));
`

Reply:
Hey Don! I think assertThat.hasSameElementsAs() reads easier for beginners, and people may not be familiar with Bag

Reply:
Adding a new dependency (with other transitive dependencies) for a single assertion method does not make sense to me. The kata is intended to teach new Java features, not to teach a third-party assertion library.  Add a method named "assertHasSameElementsAs(List, List)" and the readability is similar. 

```
public void assertHasSameElementsAs(List<?> expected, List<?> actual)
{
    assertEquals(Bags.mutable.of(expected), Bags.mutable.of(actual));
}
```

Reply:
I agree with @emilie-robichaud, Bag is an advanced topic and Eclipse Collections is itself not covered as part of this kata and it would not make sense to add it in tests just for assertions.

Reply:
@prathasirisha @emilie-robichaud The method assertHasSameElementsAs above can be implemented using JDK collections only then.